### PR TITLE
dev-python/jaraco-collections: bump to 1.5

### PR DESCRIPTION
**Pull request by sbraz**
https://github.com/jaraco/jaraco.collections/compare/1.4.1...1.5
No new deps, it's a trivial bump.

### CODE REVIEW

File: dev-python/jaraco-collections/jaraco-collections-1.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		${RDEPEND}
+		>=dev-python/pytest-2.8[${PYTHON_USEDEP}]
+		dev-python/pytest-runner[${PYTHON_USEDEP}]
+	)
+"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+python_prepare_all() {
+	if use test && ! has_version "dev-python/jaraco-text"; then
```
Comment (monsieurp) [inline dev-python/jaraco-collections/jaraco-collections-1.5.ebuild]: I don't understand this if clause. Why don't you declare `dev-python/jaraco-text` as a dependency in `test?`?

Comment (gktrk) [inline dev-python/jaraco-collections/jaraco-collections-1.5.ebuild]: There's a circular dependency between them

**Comment by gktrk** [comment]
@gentoo/proxy-maint

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/25d9a6c2b75be39cc49d3c588f9851bff8baf47d...775fdc6f62842368530e83275372113c9073ad64. Thanks!

**Pull request closed without merging**